src/main/java/de/unistuttgart/iste/gits/flashcardservice/persistence/mapper/FlashcardMapper.java

```diff
@@ -0,0 +1,31 @@
+package de.unistuttgart.iste.gits.flashcardservice.persistence.mapper;
+import de.unistuttgart.iste.gits.generated.dto.Flashcard;
+import de.unistuttgart.iste.gits.generated.dto.CreateFlashcardInput;
+import de.unistuttgart.iste.gits.generated.dto.UpdateFlashcardInput;
+import de.unistuttgart.iste.gits.generated.dto.CreateFlashcardSetInput;
+import de.unistuttgart.iste.gits.flashcardservice.persistence.dao.FlashcardEntity;
+import lombok.RequiredArgsConstructor;
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Component;
+@Component
+@RequiredArgsConstructor
+public class FlashcardMapper {
+
+    private final ModelMapper modelMapper;
+    public FlashcardMapper(ModelMapper modelMapper) {
```

Comment:
Also here please remove the duplicated constructor and format the file correctly